cider/exec/module/CiderRuntimeModule.cpp

```diff
@@ -608,6 +608,9 @@ CiderRuntimeModule::fetchResults(int32_t max_row) {
     auto out_batch = setSchemaAndUpdateAggResIfNeed(std::move(non_groupby_agg_result));
     // reset need to be done after data consumed, like bitmap for count(distinct)
     resetAggVal();
+    if (ciderCompilationOption_.use_cider_data_format) {
```

Comment:
I don't think we should do this. `convertToArrowRepresentation` method should be removed and just for test usage.

Reply:
Done. Remove convert function code to CiderBatchUtils.cpp.